### PR TITLE
Set DEFAULT_LIMITFREERELAY = 0 kB/minute

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

**Comment by Mirobit** [comment]
ACK
My nodes get flooded with these 2 year old spam transactions after every block:
https://www.blocktrail.com/BTC/tx/27cdfec78b2e0e97762a8486d5440bfc02dac3c2cd278ac803b81062c7212332

**Comment by sdaftuar** [comment]
Concept ACK.  

Is it worth warning users of the parameter interaction if they are setting `-relaypriority=0`?

**Comment by MarcoFalke** [comment]
> Is it worth warning users of the parameter interaction if they are setting -relaypriority=0?

Suggestions welcome, but I am not sure if this is required. I understand the relaypriority setting is orthogonal to the limitfreerelay setting. If the user set `-relaypriority=0` today, they'd still see their transaction rate limited (with 15 kB/min). So when they depend on free relay, they most likely adjusted their `-limitfreerelay` setting anyway.

File: doc/release-notes.md [hunk 1]
```diff
@@ -53,11 +53,15 @@ Removal of Priority Estimation
...
   number of blocks has been removed.  The rpc calls are deprecated and will either
-  return -1 or 1e24 appropriately. The format for fee_estimates.dat has also
+  return -1 or 1e24 appropriately. The format for `fee_estimates.dat` has also
   changed to no longer save these priority estimates. It will automatically be
   converted to the new format which is not readable by prior versions of the
   software.
 
+- The concept of "priority" transactions is planned to be removed in the next
+  major version. To prepare for this, the default for the rate limit of priority
+  transactions (`-limitfreerelay`) has been set to `0` kB/minute.
```
Comment (luke-jr) [inline doc/release-notes.md]: NACK. There is no plan to remove priority transactions, only "smart priority" attempting to use it. Don't confuse priority with gratis txns... they are different.

**Comment by sipa** [comment]
@luke-jr Sorry, but we're planning to remove priority. It is no longer useful to maintain.

**Comment by laanwj** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/9179/commits/fa7cc5a508921182a4ebf1f9ed296664e5044007

**Comment by luke-jr** [comment]
@sipa It remains useful and used. NACK removing it from mining.

**Pull request merged**